### ISSUE TITLE
Spark last aggregate function failed in fuzzer test.

### ISSUE THREAD

**Issue by Yohahaha** [facebookincubator/velox#4482]
### Description

PR #4467  will add SparkAggregateFuzzerTest, but last function failed currently.

### Error Reproduction

/mnt/disk1/meta-velox/debug-ali3-hz/velox/exec/tests/spark_aggregation_fuzzer_test --seed 123 --steps 10 --logtostderr=1 --minloglevel=0 --repro_persist_path=/tmp/aggregate_fuzzer_repro --only=last

### Relevant logs

```bash
I0401 12:37:46.031320 741427 AggregationFuzzer.cpp:251] Total functions: 1 (2 signatures)
I0401 12:37:46.031486 741427 AggregationFuzzer.cpp:253] Functions with at least one supported signature: 1 (100.00%)
I0401 12:37:46.031494 741427 AggregationFuzzer.cpp:257] Functions with no supported signature: 0 (0.00%)
I0401 12:37:46.031497 741427 AggregationFuzzer.cpp:259] Supported function signatures: 2 (100.00%)
I0401 12:37:46.031499 741427 AggregationFuzzer.cpp:263] Unsupported function signatures: 0 (0.00%)
I0401 12:37:46.031505 741427 AggregationFuzzer.cpp:532] ==============================> Started iteration 0 (seed: 123)
I0401 12:37:46.036968 741427 AggregationFuzzer.cpp:624] Executing query plan: 
-- Aggregation[SINGLE [g0, g1] a0 := last(ROW["c0"],ROW["c1"])] -> g0:TINYINT, g1:VARCHAR, a0:TIMESTAMP
  -- Values[1000 rows in 10 vectors] -> c0:TIMESTAMP, c1:BOOLEAN, g0:TINYINT, g1:VARCHAR
E0401 12:37:46.037398 741427 Exceptions.h:68] Line: ../velox/functions/sparksql/aggregates/LastAggregate.cpp:582, Function:operator(), Expression:  Unknown input type for last aggregation TIMESTAMP, Source: RUNTIME, ErrorCode: INVALID_STATE
E0401 12:37:46.037592 741427 Exceptions.h:68] Line: ../velox/exec/Task.cpp:1465, Function:terminate, Expression:  Cancelled, Source: RUNTIME, ErrorCode: INVALID_STATE
I0401 12:37:46.037617 741427 Task.cpp:1473] Terminating task test_cursor 1 with state Canceled after running for 0 ms.
W0401 12:37:46.037659 741427 VeloxPromise.h:40] PROMISE: Unfulfilled promise is being deleted. Context: 
I0401 12:37:46.038033 741427 AggregationFuzzer.cpp:452] Persisted input: /tmp/aggregate_fuzzer_repro/velox_vector_JXZBtr and plan: /tmp/aggregate_fuzzer_repro/velox_plan_fJcoDq
terminate called after throwing an instance of 'facebook::velox::VeloxRuntimeError'
  what():  Exception: VeloxRuntimeError
Error Source: RUNTIME
Error Code: INVALID_STATE
Reason: Unknown input type for last aggregation TIMESTAMP
Retriable: False
Function: operator()
File: ../velox/functions/sparksql/aggregates/LastAggregate.cpp
Line: 582
```

**Comment by kagamiori** [comment]
Hi @Yohahaha, I saw you closed the related issue #4485. Has this issue #4482 been fixed or is there more work needs to be done?

**Comment by Yohahaha** [comment]
Hi @kagamiori 

Some functions accept a bool config in their signatures, #4485 wants to support these functions in fuzzer test, but I'm not sure that issue is necessary.

I could change last function signatures registration to make it works well with fuzzer test to fix #4482,  but the `ignoreNull` param has to be removed due to #4485.

So, these issues still not been fixed.

**Comment by kagamiori** [comment]
Hi @Yohahaha, Thank you for sharing the context! Yes, we should make fuzzer support functions that requires constant inputs like the "last" function. I'm seeing that LastAggregate only requires `ignoreNullVector.isConstantMapping()` (i.e., requiring that input to be a ConstantVector). Does VectorFuzzer::fuzzConstant() works for this case?
https://github.com/facebookincubator/velox/blob/1e7a8907ab32470db574d22488ca986bdd7cb0cd/velox/vector/fuzzer/VectorFuzzer.h#L189-L192

**Comment by Yohahaha** [comment]
Hi @kagamiori 

We should use different function name to distinguish whether ignore null, seems no one wants to use direct SQL on Velox except UT.

I have fixed this problem in PR #4578.
